test/test-os-version.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+// Testcase for validating reported OS version
+
+const common = require('./common.js');
+const nodereport = require('../');
+const os = require('os');
+const tap = require('tap');
+
+const os_map = {
+  'aix': 'AIX',
+  'darwin': 'Darwin',
+  'linux': 'Linux',
+  'win32': 'Windows',
+};
+const os_name = os_map[os.platform()];
+const report_str = nodereport.getReport();
+const version_str = report_str.match(/OS version: .*(?:\r*\n)/);
+if (common.isWindows()) {
+  tap.match(version_str, new RegExp('OS version: ' + os_name), 'Checking OS version');
+} else if (common.isAIX() && !os.release().includes('.')) {
```

Comment:
Sorry, you're right. I think it's `Array.includes()` that isn't there in v4, `string.includes()` is.